addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SourceCodeDisclosureWebInfScanRule.java

```diff
@@ -344,4 +346,9 @@ public int getCweId() {
     public int getWascId() {
         return 34; // Predictable Resource Location
     }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(buildWebinfAlert("class A\n{\n}\n").build());
+    }
```

Comment:
I guess I should alert ref this one and include both?

Reply:
Makes sense.